src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -187,6 +187,14 @@ public void testEmptyBlock() throws Exception {
         testByConfig("EmptyBlock/context/defaultContextConfig.xml");
     }
 
+    @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/184")
```

Comment:
Looks like incorrect link to a closed issue. Please put here a link to that issue where this ignore will be fixed.

Reply:
done, removed.